apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -1006,11 +1000,10 @@ update_authenticator(ConfKeyPath, ChainName, AuthenticatorID, Config) ->
         )
     of
         {ok, #{
-            post_config_update := #{emqx_authentication := #{id := ID}},
-            raw_config := AuthenticatorsConfig
+            post_config_update := #{emqx_authentication := #{id := _ID}},
```

Comment:
nit: these two lines can be deleted

Reply:
Thought I'd keep it as a sort of sanity check. But yeah, just looking at `ok` should do it.